src/lib/components/EventListElement/index.svelte

```diff
@@ -0,0 +1,72 @@
+<script>
+	import Icon from '$lib/components/Icon/index.svelte';
+	export let title,
+		date,
+		url = '';
+	export let isPast = false;
+
+	// function that formats the standard ISO formatted date param in a better to read date format
+	function formatDate(inputDate) {
+		let rawDate = new Date(inputDate);
+		let formattedDate = rawDate.toDateString();
+		return formattedDate;
+	}
+
+</script>
+
+<figure class="event-tile">
+	{#if isPast === true}
+		<span class="past-event">Past event</span>
+	{/if}
+	<h2><a href={url}> {title} </a></h2>
+	<p>
+		<span class="icon-wrapper"
+			><Icon name="calendar" width="25px" height="25px" />{formatDate(date)}</span
+		>
```

Comment:
this is kind of a weird formatting. why not do this:

```
		<span class="icon-wrapper">
			<Icon name="calendar" width="25px" height="25px" />{formatDate(date)}
		</span>
```

Reply:
👍